### DISCUSSION TITLE
Need document to integrate Kafka to Logstash

### DISCUSSION THREAD

**Discussion by praveensams16** [strimzi/strimzi-kafka-operator#8085] [General]
Hi Team , 

  I like to integrate strimzi kafka to logstash with Mutual TLS , I need following parameter to pass in logstash to connect strimzi 
  
  It would be helpful if I get document fot that 
  
  
  
<img width="595" alt="image" src="https://user-images.githubusercontent.com/14541543/217924630-1e31b4b0-64e0-4fb8-b0a4-62e32dbc1cf9.png">

**Comment by scholzj** [comment]
Did you check the Strimzi and Kafka docs? I think there are several sections describing how to connect clients using mTLS. With a little bit of trying, you should be able to get Logstash working with it - the option names are mostly similar.
